copilot/src/App/App.js

```diff
@@ -1,28 +1,129 @@
 import AgateDecorator from '@enact/agate/AgateDecorator';
-import kind from '@enact/core/kind';
+import openSocket from 'socket.io-client';
 import qs from 'query-string';
+import {Cell, Row} from '@enact/ui/Layout';
+import kind from '@enact/core/kind';
 import React from 'react';
-
+import Button from '@enact/agate/Button';
 import css from './App.less';
+import Popup from '@enact/agate/Popup';
 
 const args = qs.parse(typeof window !== 'undefined' ? window.location.search : '');
```

Comment:
I think this can be deleted (and the import for `qs`....maybe we can remove the dependency, as well?)